src/utilities/time.ts

```diff
@@ -85,9 +85,9 @@ export function convertDoyToYmd(doyString: string, includeMsecs = true): string
         padStart(`${date.getUTCDate()}`, 2, '0'),
       ].join('-')}T${parsedDoy.time}`;
       if (includeMsecs) {
-        return ymdString;
+        return ymdString + 'Z';
```

Comment:
super pedantic (feel free to veto), do you mind changing this to use template strings to be consistent with the style of the function?

Reply:
Done!